authgear/values.yaml

```diff
@@ -5,6 +5,9 @@ authgear:
   databaseSchema: public
 
   redisURL: redis://localhost
+  redisAutoRewriteAOF:
```

Comment:
Rename this to

```
redisAutoRewriteAOFCronjob:
  enabled: false
  schedule: "0 3 * * *"
  # Use 6 and set imagePullPolicy to always so that we always use the latest version of 6.
  image: redis:6
  resources:
    requests:
      cpu: 100m
      memory: 32Mi
    limits:
      cpu: 200m
      memory: 64Mi
```